schemas/team_detail.yaml

```diff
@@ -0,0 +1,54 @@
+---
+team_detail:
```

Comment:
I don't see any difference with `team`, please use `team` instead and remove this file.

Reply:
it is within the user. team details adds more info

Reply:
In that case, team.yml is wrong and should point to user_preview.